docs/api/version-history.md

```diff
@@ -87,6 +87,10 @@ keywords: "API, Docker, rcli, REST, documentation"
   volume (CNI). This option can only be used if the daemon is a Swarm manager.
   The Volume response on creation now also can contain a `ClusterVolume` field
   with information about the created volume.
+* The `BuildCache.Parent` field, as returned by `GET /system/df` is deprecated
+  and is now omitted. API versions before v1.42 continue to include this field.
+* `GET /system/df` now includes a new `Parents` field, for "build-cache" records,
+  which contains a list of parent ID's for the build-cache record.
```

Comment:
I believe it is `IDs`

Reply:
Yes, I think you're right; must've been in "dutch" mode while I wrote this 🇳🇱 😁

Reply:
fixed 👍